.github/workflows/CI.yml

```diff
@@ -191,7 +191,7 @@ jobs:
           profile: minimal
           toolchain: stable
           override: true
-      - run: cargo build --features=c-interface
+      - run: make
```

Comment:
Thanks for this change. It'll allow the CI to finally check the run of the example too. I'm ashamed for not changing it before...